settings/defaults.go

```diff
@@ -58,6 +58,7 @@ func defaultSettings() Settings {
 			"SNK - Neo Geo Pocket":                           "mednafen_ngp_libretro",
 			"Sony - PlayStation":                             playstationCore,
 		},
+		FileDirectory:        xdg.DataHome,
```

Comment:
When I first launched ludo on this branch, it looks like FileDirectory defaulted to `~/.local/share`. This is not a sane default value for the rom scanner. Let's use `usr.HomeDir`.

Reply:
I had misunderstood where `xdg.DataHome` was pointing to, we could switch it back to `usr.HomeDir` or `xdg.DownloadDir` which would default to `~/Downloads` on unix-likes or `%USERPROFILE%\Downloads` on windows. I'll point it to `usr.HomeDir` for the time being =)